### ISSUE TITLE
How to modify the database using variable ? 

### ISSUE THREAD

**Issue by weezixx** [2shady4u/godot-sqlite#24]
Hi ! 

I'm making a game with a lot of words. 

The "game" for the moment is to select a random line, and the player has to select the correct translation. 

If the player is right, I need to change a value in my db based on the random line. 

Ex : randomline = 5

      db.query("UPDATE words SET trials = 1 WHERE id = ?",randomline)

How may I do that with your module ? 

Here my request on the website : [https://godotengine.org/qa/91156/sqlite-cant-access-data-with-variable](url)

Thanks a lot

**Comment by 2shady4u** [comment]
Hello!

Parameter bindings are coming in the next major release in which you'll hopefully be able to do that exact thing.
At the moment however, you should be able to do it yourself by modifying your query as such:

```gdscript
db.query("UPDATE words SET trials = 1 WHERE id = " + str(randomline) + ";")
```

or equivalently:

```gdscript
db.query("UPDATE words SET trials = 1 WHERE id = {0};".format([randomline]))
```

**Comment by weezixx** [comment]
Amazing ! This is my X-mas gift ! 

Thank you very much ! 

Stay safe

**Comment by 2shady4u** [comment]
Behaviour in upcoming major release should behave like this:

```gdscript
db.query_with_bindings("UPDATE words SET trials = 1 WHERE id = ?;", [randomline])
```

(This is a brand-new function that is not available in the current version)

You can check out the beginnings of this release here:
https://github.com/2shady4u/godot-sqlite/tree/feature-add-blobs

**Comment by 2shady4u** [comment]
Implemented/resolved in version 2.1 as found in the [releases](https://github.com/2shady4u/godot-sqlite/releases/tag/v2.1) tab.
